setup.py

```diff
@@ -35,8 +47,13 @@ def _read(fname):
     ),
     url="https://github.com/0soft/graphene-django-plus",
     packages=["graphene_django_plus"],
-    setup_requires=["pytest-runner >=4.0"] if _need_pytest else [],
-    long_description=_read("README.md"),
+    setup_requires=["pytest-runner"],
+    tests_require=tests_require,
+    extras_require={
+        "test": tests_require,
```

Comment:
I would only change this to `test_requires` because the other variable is called `dev_requires` (with the `s` at the end)